assets/src/dashboard/animations/effects/flyIn/index.js

```diff
@@ -23,7 +23,7 @@ import { DIRECTION } from '../../constants';
 
 export function EffectFlyIn({
   duration = 500,
-  direction = DIRECTION.TOP_TO_BOTTOM,
+  flyInDir = DIRECTION.TOP_TO_BOTTOM,
```

Comment:
Typo, I forgot `direction` is an actual Web API parameter, so I renamed this to `flyInDir` and I did the same with `FloatOn` too (these were the only two animations that use `DIRECTION`).